### PR TITLE
Move pallet-bridge-dispatch types to primitives

### PR DESCRIPTION
**Pull request by fewensa**
issue: https://github.com/paritytech/parity-bridges-common/issues/895

### CODE REVIEW

File: primitives/message-dispatch/Cargo.toml [hunk 1]
```diff
@@ -10,9 +10,20 @@ license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
 bp-runtime = { path = "../runtime", default-features = false }
 codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false }
 
+# Substrate Dependencies
+
+frame-support = { git = "https://github.com/paritytech/substrate", branch = "master" , default-features = false }
+frame-system = { git = "https://github.com/paritytech/substrate", branch = "master" , default-features = false }
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "master" , default-features = false }
```
Comment (HCastano) [inline primitives/message-dispatch/Cargo.toml:17]: You don't need these two dependencies here

```suggestion
```

**Review by HCastano — APPROVED** [review]
Look good aside from the extra deps!

File: primitives/message-dispatch/Cargo.toml (hunk 1 above)
Comment (fewensa) [reply primitives/message-dispatch/Cargo.toml]: done it

**Comment by HCastano** [comment]
Thanks!

**Pull request merged**